ampd/README.md

```diff
@@ -108,6 +108,19 @@ cosmwasm_contract = 'axelar14lh98gp06zdqh5r9qj3874hdmfzs4sh5tkfzg3cyty4xeqsufdjq
 chain_name = 'avalanche'
 chain_rpc_url = "https://api.avax-test.network/ext/bc/C/rpc"
 
+[[handlers]]
+type = "SolanaMsgVerifier"
+cosmwasm_contract = "axelar1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqecnww6"
+chain_name = "solana"
+chain_rpc_url = "https://api.devnet.solana.com"
+max_tx_cache_entries = 6
```

Comment:
Is this correct? Where is this handled?

Reply:
This was part of a previous, already removed implementation. Fixed at https://github.com/axelarnetwork/axelar-amplifier/pull/744/commits/a28db9d4d29ee5ed18c9d585664ab6468cb4084e by removing it.